server/services/schema.spec.ts

```diff
@@ -0,0 +1,63 @@
+import '../utils/mockMongo'
+import { createSchema, findSchemaByRef, findSchemasByUse, serializedSchemaFields } from './schema'
+import SchemaModel from '../models/Schema'
+
+describe('test schema service', () => {
+
+  const uploadSchema: any = {
+    name: 'upload-schema',
+    reference: 'upload',
+    use: 'UPLOAD',
+    schema: {}
+  }
+  const uploadSchema2: any = {
+    name: 'upload-schema2',
+    reference: 'upload2',
+    use: 'UPLOAD',
+    schema: {}
+  }
+  const deploymentSchema: any = {
+    name: 'deployment-schema',
+    reference: 'deployment',
+    use: 'DEPLOYMENT',
+    schema: {}
+  }
+
+  beforeEach(async () => {
+    const uploadDoc = new SchemaModel(uploadSchema)
+    await uploadDoc.save()
+    const uploadDoc2 = new SchemaModel(uploadSchema2)
+    await uploadDoc2.save()
+    const deploymentDoc = new SchemaModel(deploymentSchema)
+    await deploymentDoc.save()
+  });
+
+  test('that the serializer returns the correct properties', () => {
+    const properties: any = serializedSchemaFields()
+    expect(properties.mandatory).toStrictEqual(['_id', 'reference', 'name', 'use'])
+  })
+
+  test('we can fetch schema by reference', async () => {
+    const fetchedSchema: any = await findSchemaByRef('upload')
+    expect(fetchedSchema).not.toEqual(null)
+    expect(fetchedSchema.name).toBe(uploadSchema.name)
+  })
+
+  test('we can find upload schemas with a limit of 1', async () => {
+    const fetchedSchema: any = await findSchemasByUse('UPLOAD', 1)
```

Comment:
No need for any here, code works without it.

Reply:
removed